comprehensive/lms/templates/navigation.html

```diff
@@ -64,7 +64,7 @@ <h1 class="logo">
         </%block>
       </a>
     </h1>
-
+    % if not enable_account_linking:
```

Comment:
we should probably leave this change out in favor of not mixing in MSA changes..
Also I am not quite sure if we have that variable available already.. we might need to check oxa-tools.